### PR TITLE
build(deps): Fix renovate lockfile maintenance

### PR DESCRIPTION
**Pull request by jacobmichels**
#### Description

I recently installed starship (at commit `f353ab4ad1a4dcc9ca9268b3db0dc271842aab10`) via `cargo install starship --locked` and received the following warning:
```
warning: package `ordered-multimap v0.7.1` in Cargo.lock is yanked in registry `crates-io`, consider running without --locked
```

I did not like to see that `starship` was being built with a yanked dependency.

I cloned `starship/starship` and ran `cargo update --dry-run` and found that there were many updates available in `cargo.lock`. I inspected starship's renovate config and saw that `lockFileMaintenance` is supposed to be enabled, which should have prevented the lockfile getting too stale, but it appears to be misconfigured. 
```
{
    ...
    "packageRules": [
        ...
        // Update (rust) lockfiles weekly
        {
            "matchDatasources": ["crate"],
            "lockFileMaintenance": {
                "enabled": true,
                "extends": ["schedule:weekly"]
            }
        }
    ],
    ...
}

```
[Renovate's docs](https://docs.renovatebot.com/configuration-options/#lockfilemaintenance) suggest configuring lockFileMaintenance differently. I modified the renovate config to enable lockFileMaintenance on a weekly schedule and have it automerge. [The renovate docs say automerging lockFileMaintenance is safe](https://docs.renovatebot.com/key-concepts/automerge/#automerge-lock-file-maintenance).

I also looked through renovate bot's PRs on this repo and verified that no lockfile maintenance PRs had been opened.

#### Motivation and Context

`lockFileMaintenance` is misconfigured, it should be fixed. 

#### How Has This Been Tested?

- [ ] I have tested using **MacOS**
- [ ] I have tested using **Linux**
- [ ] I have tested using **Windows**

I am unsure how to test this.

#### Checklist:

- [ ] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.

### CODE REVIEW

File: .github/renovate.json5 [hunk 1]
```diff
@@ -16,6 +16,12 @@
     "semanticCommitType": "build",
     "rangeStrategy": "bump",
     "rebaseWhen": "conflicted",
+    "lockFileMaintenance": {
+        "enabled": true,
+        "extends":["schedule:weekly"],
```
Comment (davidkna) [inline .github/renovate.json5:21]: Style:
```suggestion
        "extends": ["schedule:weekly"],
```

**Review by davidkna — COMMENTED** [review]
I would have preferred to only update the cargo lockfiles and avoid bumping the npm lockfile for the docs.
But since only updating that isn't currently possible and since the docs have a basic build-check in CI now, this should be fine.

**Comment by jacobmichels** [comment]
I'm unsure why the `config schema up-to-date` check is failing. I would attempt a fix, but the warnings give me no lead. Merging master in didn't fix.

**Review by davidkna — APPROVED** [review]
LGTM

**Comment by andytom** [comment]
Thank you for your contribution @jacobmichels and thank you for reviewing @davidkna.

**Pull request merged**